meteor-addons.js

```diff
@@ -1,13 +1,15 @@
 // Meteor specific addons to the bootbox.js object
 bootbox.templateDialog = function(options) {
-  var div = document.createElement("div");
-  UI.insert(UI.renderWithData(options.template, options.data), div);
-  options.message = div;
+
+  var view = Blaze.renderWithData(options.template, options.data);
+
+  options.message = Blaze.toHTML(view);
```

Comment:
We don't want to use Blaze.toHTML here because it looses the fine-grained reactivity which is the goal of bootbox.templateDialog.

Instead we should keep `div = document.createElement` and use `Blaze.renderWithData(options.template, options.data, div)`